apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -480,12 +480,22 @@ defmodule OMG.Watcher.ExitProcessor.Core do
       get_invalid_ife_challenges(request, state)
       |> Enum.map(fn txbytes -> %Event.InvalidIFEChallenge{txbytes: txbytes} end)
 
+    available_piggybacks_events =
```

Comment:
at this point, we need to acknowledge that the `determine_blocks_to_get` must return blocks that satisfy a set of criteria. Let's document them, in `@doc` of `determine_blocks_to_get`:

```
1/ blocks where any input to any IFE was spent
2/ blocks where any output to any IFE was spent (CC @paulperegud is this true?)
3/ blocks where the whole IFE transaction **might've** been included, to get piggyback availability and to get InvalidIFEChallenge's
```

Is this list accurate and complete though? 

The reason why I'd like this doc is that we don't want for someone to optimize this blindly, and break one of the required "block consumers"

Reply:
@paulperegud 
`2/ blocks where any output to any IFE was spent (CC @paulperegud is this true?)`

Reply:
BTW I think my comment is obsolete, having worked on `block getting` Pepesza's doing for a while. Nevermind this and sry